api/app/signals/apps/questionnaires/rest_framework/views/public/questions.py

```diff
@@ -83,3 +84,6 @@ def answer(self, request, *args, **kwargs):
         data.update({'next_question': next_question_data})
 
         return Response(data, status=201)
+
+    def list(self, request, *args, **kwargs):
```

Comment:
Please remove this function and use `HALViewSetRetrieve` instead of `DatapuntViewSet` instead.
Example:

```
class PublicQuestionnaireViewSet(HALViewSetRetrieve):
    lookup_field = 'uuid'
    lookup_url_kwarg = 'uuid'
```

instead of 

```
class PublicQuestionnaireViewSet(DatapuntViewSet):
    lookup_field = 'uuid'
    lookup_url_kwarg = 'uuid'
```

Reply:
Done, I left the test that checks whether we can list questionnaires or questions so this does not get re-enabled.